test/DebugInfo/macro.swift

```diff
@@ -0,0 +1,21 @@
+// REQUIRES: objc-interop
```

Comment:
Why does this require `objc-interop`?

Reply:
I think you are right, this might work with C-interop alone, too.